spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/server/db/migration/AbstractSmokeTest.java

```diff
@@ -50,11 +49,12 @@
  *
  * @author Corneil du Plessis
  */
-@SpringBootTest(classes = {DataFlowServerApplication.class}, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-@TestPropertySource(properties = {
-	"spring.jpa.hibernate.ddl-auto=none"
-})
+@SpringBootTest(classes = DataFlowServerApplication.class,
+		webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+		properties = "spring.jpa.hibernate.ddl-auto=none")
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_CLASS)
```

Comment:
Without this, each subsequent concrete test would still be around in the context and the count of task executions would increase by 2 on each test run. Another indicator this test has not been running. 

Example failed run can be seen [here](https://github.com/spring-cloud/spring-cloud-dataflow/actions/runs/7092775067/job/19304786985) but you will need to download the raw log to see it.